src/invoices/InvoiceDetails/InvoiceDetails.js

```diff
@@ -98,6 +102,14 @@ class InvoiceDetails extends Component {
       />
     );
 
+    const viewVoucherButton = (
+      <Button
+        data-test-view-voucher-button
```

Comment:
@Feboll @AndreiShumski2 
Guys, let's check out common styles to be consistent with the rest of FOLIO devs.
https://wiki.folio.org/display/FOLIJET/Code+style+rules+not+covered+by+eslint
cc @NikitaSedyx

Reply:
Put it on a new line for future, I'll add handler in the next my PR